### PR TITLE
204 add slack channel name on create new project form

### PR DESCRIPTION
**Pull request by chris110408**
Issue addressed
Add form field to add slack channel name on Create New Project form issue 204

fixes #
Expected Behavior
Add Form input field to add slack_channel_name
Add a label next to the Input field displaying slack_channel_name
Current Behavior
There is no form input field for slack channel name.

Possible Solution
Add Semantic UI react Form.Input to add Input to add field to the form

test:
the original unit test for action has problem use the newproject data to compare with newproject and always show success.

### CODE REVIEW

File: src/tests/actions/createProjectAction.test.js [hunk 1]
```diff
@@ -5,15 +5,16 @@ import configureMockStore from 'redux-mock-store'
 import { CREATE_PROJECT } from '../../types'
 import newProject from '../../../cypress/fixtures/newlyCreatedProject'
 
-describe('createProject', () => {
+describe.only('createProject', () => {
```
Comment (joaopapereira) [inline src/tests/actions/createProjectAction.test.js:8]:
```suggestion
describe('createProject', () => {
```

File: src/tests/actions/createProjectAction.test.js [hunk 2]
```diff
@@ -25,17 +26,21 @@ describe('createProject', () => {
     moxios.uninstall()
   })
 
-  it('posts project info to Rails backend and returns without error', async () => {
+  it.only('posts project info to Rails backend and returns without error', async () => {
```
Comment (joaopapereira) [inline src/tests/actions/createProjectAction.test.js:29]:
```suggestion
  it('posts project info to Rails backend and returns without error', async () => {
```

File: src/tests/components/ProjectForm.test.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import { shallow } from 'enzyme'
-import { ProjectForm } from '../../components/ProjectForm'
+import ProjectForm from '../../components/ProjectForm'
 
-describe('ProjectForm', () => {
+describe.only('ProjectForm', () => {
```
Comment (joaopapereira) [inline src/tests/components/ProjectForm.test.js:5]:
```suggestion
describe('ProjectForm', () => {
```

File: src/tests/components/ProjectForm.test.js [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
   let wrapper
   let props = {
     handleSubmit: jest.fn()
   }
 
   beforeEach(() => {
+    console.log(ProjectForm)
```
Comment (joaopapereira) [inline src/tests/components/ProjectForm.test.js:12]:
```suggestion
```

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Great contribution. Looks good, but please review my comments

**Comment by mattwr18** [comment]
closed in favor of https://github.com/AgileVentures/WebsiteOne-FE/pull/243, which was also put in by @chris110408

**Pull request closed without merging**